ext/com_dotnet/com_handlers.c

```diff
@@ -615,6 +615,10 @@ void php_com_object_free_storage(zend_object *object)
 		zend_hash_destroy(obj->id_of_name_cache);
 		FREE_HASHTABLE(obj->id_of_name_cache);
 	}
+
+	if (UNEXPECTED(GC_FLAGS(object) & IS_OBJ_WEAKLY_REFERENCED)) {
+		zend_weakrefs_notify(object);
+	}
```

Comment:
I think it should be fine call the std obj dtor here instead.

Reply:
Done, I left the function exported though, because of strange things that might be in the wild ... I could be convinced that's not absolutely necessary: anything that calls init should call dtor, but since we have core exts that didn't, there may be exts in the wild that don't also ...